diavlos/src/diaugeia/diaugeia.py

```diff
@@ -0,0 +1,56 @@
+"""A module for fetching diaugeia API information."""
+
+import logging
+import requests
+
+from .error import DiaugeiaErrorCode as ErrorCode
+
+logger = logging.getLogger(__name__)
+
+API_BASE_URL = "https://diavgeia.gov.gr/opendata"
+API_TEST_URL = "https://test3.diavgeia.gov.gr/luminapi/opendata/decisions/"
+
+
+class DiaugeiaError(Exception):
+    """diaugeiaError exception"""
+
+
+def _error(message):
+    logger.error(message)
+    raise DiaugeiaError(message)
+
+
+def _request_data(code):
```

Comment:
There is no real need for this function, since it's called no more than 1 times.